storage/local/chunk/chunk.go

```diff
@@ -402,23 +409,23 @@ func newIndexAccessingChunkIterator(len int, acc indexAccessor) *indexAccessingC
 	return &indexAccessingChunkIterator{
 		len:       len,
 		pos:       -1,
-		lastValue: ZeroSamplePair,
+		lastValue: model.SamplePair{Timestamp: model.Earliest},
```

Comment:
Is this ok or should we define `ZeroSamplePair` in a place accessible by both packages?